src/libraries/System.Security.Principal.Windows/tests/SecurityIdentifierTests.cs

```diff
@@ -9,6 +9,43 @@
 
 public class SecurityIdentifierTests
 {
+
+    [Fact]
+    public void SecurityIdentifierBinaryCtorThrowsOnNull()
+    {
+        Assert.Throws<ArgumentNullException>("binaryForm", () => new SecurityIdentifier(null, 0));
```

Comment:
```suggestion
        AssertExtensions.Throws<ArgumentNullException>("binaryForm", () => new SecurityIdentifier(null, 0));
```

In 4 pages of results for github's fuzzy search on `Assert.Throws<ArgumentNullException>` all of the ones supplying a parameter name are using AssertExtensions.  So let's use that one for consistency.

Reply:
Done.